### ISSUE TITLE
Display Image From database

### ISSUE THREAD

**Issue by m-henderson** [RTask/RTask#21]
I tried to make it where the user could see the image that was uploaded with the ticket when viewing a ticket but I cant figure out how to get the string to show correctly in base64 format. If you guys suggest another way of displaying the uploaded image just let me know because I would be interested. 

Thanks (:

**Comment by ashvinimore** [comment]
Hello Sir,
So Do you want to store the image in sqlite db in blob or base64 format .
and display that image into the frontend ?
Thank You

**Comment by m-henderson** [comment]
Can we store it in MySQL? Blob is fine or we can store the path and keep file on server

**Comment by ashvinimore** [comment]
Yes Sir. Path should be fine. We store path and keep file on server. and provide path to  the frontend from views.
Is it ok?

**Comment by m-henderson** [comment]
@ashvinimore that should work.

**Comment by ashvinimore** [comment]
Hello Sir,
I have created imagefield which browse path and save the url into db.
and display into the admin page with record. 
Is it OK?
Thank You
